themes/aframe/layout/partials/search.ejs

```diff
@@ -1,14 +1,16 @@
 <!-- Algolia search. -->
 <script type="text/javascript" src="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.js"></script>
 <script type="text/javascript">
-  docsearch({
+  var searchConfig = {
     apiKey: '9e4a51a1da70daf959bd4750a3fa8f52',
     indexName: 'aframe',
-    inputSelector: '#search',
     algoliaOptions: {
       facetFilters: ['version:<%- docs_active_version(page) || config.aframe_version %>'],
       hitsPerPage: 10
     },
     debug: false
-  });
+  };
+
+  docsearch(Object.assign(searchConfig, {inputSelector: '.docs-search-sidebar .search-field'}));
```

Comment:
Will it apply to both search fields if we just specify a class `.search-field`? Or do we have to register twice separately.